gen/dcos-config.yaml

```diff
@@ -379,6 +379,31 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
+  - path: /etc/dcos-metrics-config.yaml
+    content: |
+      #######################################
+      # Configuration file for dcos-metrics #
+      #######################################
+
+      # Collector configuration
+      collector:
+        http_profiler: false
+
+        node:
+          poll_period: 15
+
+        # DC/OS (Mesos) agent configuration
+        mesos_agent:
```

Comment:
In master mode it only exposes host-level metrics, so it does not make requests to mesos-agent API, and therefore ignores this configuration if present.